base/cvd/cuttlefish/host/commands/cvd/unittests/selector/instance_database_test.cpp

```diff
@@ -273,23 +273,23 @@ TEST_F(CvdInstanceDatabaseTest, FindByInstanceId) {
   }
   // The end of set up
 
-  auto result1 = db.FindInstance(Query(kInstanceIdField, std::to_string(1)));
-  auto result10 = db.FindInstance(Query(kInstanceIdField, std::to_string(10)));
-  auto result7 = db.FindInstance(Query(kInstanceIdField, std::to_string(7)));
-  auto result11 = db.FindInstance(Query(kInstanceIdField, std::to_string(11)));
-  auto result3 = db.FindInstance(Query(kInstanceIdField, std::to_string(3)));
-  auto result_invalid = db.FindInstance(Query(kInstanceIdField, std::to_string(20)));
+  auto result1 = db.FindInstanceWithGroup(Query(kInstanceIdField, std::to_string(1)));
+  auto result10 = db.FindInstanceWithGroup(Query(kInstanceIdField, std::to_string(10)));
+  auto result7 = db.FindInstanceWithGroup(Query(kInstanceIdField, std::to_string(7)));
+  auto result11 = db.FindInstanceWithGroup(Query(kInstanceIdField, std::to_string(11)));
+  auto result3 = db.FindInstanceWithGroup(Query(kInstanceIdField, std::to_string(3)));
+  auto result_invalid = db.FindInstanceWithGroup(Query(kInstanceIdField, std::to_string(20)));
```

Comment:
nit: can these numbers be string constants?

Reply:
Done